railties/test/application/configuration_test.rb

```diff
@@ -1209,6 +1209,15 @@ def create
       assert_equal false, ActionController::Parameters.action_on_unpermitted_parameters
     end
 
+    test "config.action_controller.default_protect_from_forgery is true by default on production" do
+      app "production"
+
+      ActionController::Base.object_id # force lazy load hooks to run
```

Comment:
This isn't necessary, since referencing `ActionController::Base ` on the next line will run the lazy load hooks. It's only needed when testing `ActionController::Parameters` options, as loading that class won't trigger the hooks.